src/ShellCheck/Analytics.hs

```diff
@@ -2856,6 +2857,7 @@ checkPipeToNowhere _ t =
         name <- getCommandBasename cmd
         guard $ name `elem` nonReadingCommands
         guard . not $ hasAdditionalConsumers cmd
+        guard . not $ name == "mv" && "i" `elem` (map snd $ getAllFlags cmd)
```

Comment:
This is also an issue with `cp` and `rm`, so maybe ``name `elem` ["cp", "rm", "mv"] && cmd `hasFlag` "i"``

Reply:
Updated, ptal